Code/Tools/FBuild/FBuildCore/Graph/CSNode.cpp

```diff
@@ -27,7 +27,8 @@ CSNode::CSNode( const AString & compilerOutput,
 				const Dependencies & inputNodes,
 				const AString & compiler,
 				const AString & compilerArgs,
-				const Dependencies & extraRefs )
+				const Dependencies & extraRefs,
+                const Dependencies & preBuildDependencies )
```

Comment:
This change had a few whitespace issues which I've resolved in the P4 mainline.  FASTBuild source uses tabs.